### ISSUE TITLE
How can I add Event?

### ISSUE THREAD

**Issue by Rauliciii** [shrikanth7698/Collapsible-Calendar-View-Android#75]
Hi,

I want to add an UI indicator to the user that there is some event taking place in a specific date (a dot or something under the date).
Does this library have that?

I tried using calendarAdapter and simply addEventTag on the collapsibleCalendar but it doesn't work

**Comment by dierivera** [comment]
Yes, using the addEventTag is the correct way. Maybe you need to change the color of the dot? I checked my code and using that method is enough

**Comment by Rauliciii** [comment]
collapsibleCalendar.addEventTag(2021, 12, 21);
so this should work. What version on the Library are you using?

**Comment by Rauliciii** [comment]
![image](https://user-images.githubusercontent.com/12424706/146839911-57121515-0d21-4a03-af06-c40a0b9bafd2.png)
I tried everything, as you can see

**Comment by Rauliciii** [comment]
Now I saw     collapsibleCalendar.addEventTag(2021, 12, 21);
does not work because Month starts from 0 so it should have been:
collapsibleCalendar.addEventTag(2021, 12, 21);

Now I'm facing the fact that the DOT is HUGE! How can I change this?

**Comment by dierivera** [comment]
I think I had to fork the repo and customized on my end. You can check my repos and use that forked version

**Comment by Rauliciii** [comment]
![image](https://user-images.githubusercontent.com/12424706/146842161-a6064438-8c1e-4ec7-a6b8-f33ec6272aca.png)
I got here but I don't know how to access the library

**Comment by dierivera** [comment]
implementation 'com.github.dierivera:Collapsible-Calendar-View-Android:9d6be9d129'  //calendar for the agenda tab

You can use it adding this in the gradle, the numbers at the end are pointing to the last commit

**Comment by Rauliciii** [comment]
This is amazing!!! Good job dierivera

**Comment by Rauliciii** [comment]
One thing tho, do you have this change made for version 1.0.2 also? I don't enjoy the swipe motion from 1.0.3
